macro/Version.py

```diff
@@ -36,16 +55,40 @@ def execute(macro, args):
                     'New in %s</span>' % version)
 
     def distro_html(distro, distros):
-        active = [distro.encode("iso-8859-1")]
-        inactive = [x.encode("iso-8859-1") for x in distros if not x == distro]
+        html = ""
```

Comment:
May be remove this line and assign it in line 80? Makes the variable have much smaller scope.